playbooks/roles/install-butterfly/files/oc-web-terminal.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+hostname=https://{{ openshift_public_hostname }}
+password={{ generic_pass }}
+username={{ generic_user }}
+begin={{ generic_count_begin }}
+count={{ generic_count }}
+pause=5
+
+# Configure openshift.
+# if ! oc whoami &>/dev/null; then
+# printf "Type the server URL that you want to log in to, followed by [ENTER]: "
+#
+# read url
+#
+# if [[ -z "$url" ]]; then
+# printf "OpenShift server URL cannot be null. Please try with a valid URL..."
+# exit 1
+# fi
+#
+# oc login "$url"
+#
+# printf "Type cluster administrator username, followed by [ENTER]: "
+#
+# read admin_user
+#
+# printf "Type cluster administrator password, followed by [ENTER]: "
+#
+# read admin_pass
+# fi
+
+for (( i = $begin; i <= $count; i++ )); do
+ oc login "$hostname" --insecure-skip-tls-verify -u $username${i} -p $password
+
+ # Gather username for project suffix
+ PRJ_SUFFIX=${ARG_PROJECT_SUFFIX:-`echo $(oc $ARG_OC_OPS whoami) | sed -e 's/[-@].*//g'`}
+
+ # # Delete user project.
+ # oc delete project ocwt-$PRJ_SUFFIX
+
+ # Create user project.
+ oc new-project ocwt-$PRJ_SUFFIX
+
+ # Deploy web terminal container.
+ oc new-app quay.io/openshiftlabs/workshop-terminal:2.4.0
```

Comment:
so in the past i've used quay.io/openshifthomeroom/workshop-terminal..need to figure out which is the maintained one.
also, when i install the terminal, i pass in an env variable so that the appropriate oc client is added. Do we want to do something similar

for e.g
oc new-app --docker-image=quay.io/openshifthomeroom/workshop-terminal --env=OC_VERSION=3.11
-- this installs the terminal with the 3.11 oc client

Reply:
Hmmm, I will look into this @gbengataylor. Did not know that you could pass the OC_VERSION param.

Reply:
Let's make this an enhancement post- merge effort.